feat(changelog): dedibox-added-dedibox-vps-are-now-in-general-availabili 2024-05-15 (#3173)
scaleway/docs-content: changelog/may2024/2024-05-15-dedibox-added-dedibox-vps-are-now-in-general-availabili.mdx

---
title: Dedibox VPS are now in General Availability! 🚀
status: added
author:
    fullname: 'Join the #dedibox channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-05-15
category: bare-metal
product: dedibox
---

Dedibox VPS (Virtual Private Servers) are now available for general use.

> Dedibox VPS offer the power and flexibility of a dedicated server within a shared hosting environment. With Dedibox VPS, you get complete control at a fraction of the cost, making them an ideal solution for individuals, businesses, and developers alike. 

Reviewer: ```suggestion
Dedibox VPS offers the power and flexibility of a dedicated server within a shared hosting environment. With Dedibox VPS, you get complete control at a fraction of the cost, making it an ideal solution for individuals, businesses, and developers alike. 
```

> Dedibox VPS offer the power and flexibility of a dedicated server within a shared hosting environment. With Dedibox VPS, you get complete control at a fraction of the cost, making them an ideal solution for individuals, businesses, and developers alike. 

Reviewer 2: ```suggestion
Dedibox VPS offers the power and flexibility of a dedicated server within a shared hosting environment. With Dedibox VPS, you get complete control at a fraction of the cost, making them an ideal solution for individuals, businesses, and developers alike. 
```

> Dedibox VPS offers the power and flexibility of a dedicated server within a shared hosting environment. With Dedibox VPS, you get complete control at a fraction of the cost, making them an ideal solution for individuals, businesses, and developers alike. 

Reviewer 2: ```suggestion
Dedibox VPS offers the power and flexibility of a dedicated server within a shared hosting environment. With Dedibox VPS, you get complete control at a fraction of the cost, making it an ideal solution for individuals, businesses, and developers alike. 
```
